code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -437,7 +437,7 @@ Congratulations! You are now trained for xenobiology research!"}
 		species = "<span clas=='notice'>[H.dna.species.name]</span>"
 		if(L.mind && L.mind.changeling)
 			species = "<span class='warning'>Changeling lifeform</span>"
-		var/obj/item/organ/gland/temp = locate() in H.internal_organs
+		var/obj/item/organ/heart/gland/temp = locate() in H.internal_organs
```

Comment:
yep, forgot they removed hearts despite not actually using the heart slot, fair enough.